source/models/relationships.md

```diff
@@ -277,31 +277,37 @@ export default DS.Model.extend({
 });
 ```
 
-When a user comments on a blogPost, we need to create a relationship between the two records. We can simply set the `belongsTo` relationship in our new comment:
+Now, suppose we want to add comments to an existing blogPost. We can do this in two ways, but for both of them, we will first need a reference to our existing blogPost as given below:
+
+```javascript
+let myBlogPost = this.get('store').peekRecord('blog-post', 1);
+```
+Now we can either set the `belongsTo` relationship in our new comment, or, update the blogPost's `hasMany` relationship. As you might observe, we don't need to set both `hasMany` and `belongsTo` for a record, ember data will do that for us.
+
+First, let's look at setting the `belongsTo` relationship in our new comment:
 
 ```javascript
-let blogPost = this.get('store').peekRecord('blog-post', 1);
 let comment = this.get('store').createRecord('comment', {
-  blogPost: blogPost
+  blogPost: myBlogPost
```

Comment:
I really like this distinction of calling it `myBlogPost`

Reply:
Thanks! :)